### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by noopeah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
 *git strategy:*  
 1) things that are *in progress of development* are to be committed and pushed into the deDICKated feature branch. After a feature is done and merged, the feature branch is to be deleted.
 2) feature branches *must* have a continious PR for reviews.
-3) *never* push and force push to master.
+3) *never* push and force push to master.  
+4) **ALL COMMIT MESSAGES _MUST_ BE PREPENDED BY** `yaaaaaaaas girrrrrrrrrrrrl!`
```
Comment (rebar-reinforced-concrete) [inline README.md:6]: But where is the "Poni s los" picture?

**Review by rebar-reinforced-concrete — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (noopeah) [reply README.md:6]: sorry, @mishaismenska. Will add it in the next release. Due to a budget restriction we were unable to source the afformentioned image from our A\V department. Will notify you as soon as the image will be available. Sorry for the issue...

Comment (rebar-reinforced-concrete) [reply README.md:6]: Please, mind discussing our budget with your PM, @noopeah. It's impossible to work in such conditions. Could you imagine writing the course project without vegetating picture?

Comment (noopeah) [reply README.md:6]: @mishaismenska, I understand the utter importance of such imagery,  as it promotes the vegetative state of the developers(i.e. us) during the development process. Despite that, I'd like to reiterate that budget restrictions caused that, not me. I'll bring that up during the catch-up meeting with the PM(@mishaismenska), DM(@noopeah).

**Pull request merged**